packages/eslint-config-airbnb/rules/react.js

```diff
@@ -91,7 +91,7 @@ module.exports = {
 
     // Validate JSX has key prop when in array or iterator
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md
-    'react/jsx-key': 'off',
+    'react/jsx-key': ['error', { checkFragmentShorthand: true }],
```

Comment:
The rule is off on purpose, because it has too many false positives.

Reply:
I am surprised by this, but ok. Then I will just continue to set these in my projects manually.

Commit b294e7cbf, which originally set this, did not elaborate the reasoning. There all missing react rules where just disabled without any explanation why they are not enabled. And searching in the issues and PRs I also found no explanation. Maybe it is helpful, to add this reasoning?

I will change this PR, so that it keeps the rule turned off but adds a reasoning.

Reply:
We don't just enable every rule; some of them are actively harmful. That commit added all the rules that were already "not on", explicitly set to off.

We don't generally explain why a rule isn't turned on - there's too many rules for that to be sustainable.

Reply:
I disagree whether explaining these rules is sustainable but you are making the calls here, so I will close this PR.

Reply:
I said "don't generally", i didn't say this PR wouldn't be accepted or not.